frontend/src/components/eMIB/EmibTabs.jsx

```diff
@@ -10,10 +9,6 @@ import Notepad from "../commons/Notepad";
 import "../../css/emib-tabs.css";
 
 class EmibTabs extends Component {
-  static propTypes = {
```

Comment:
don't you also need to remove this from render?

Reply:
Fixed in merge conflicts commit #https://github.com/code-for-canada/project-thundercat/pull/122/commits/86955f686dc53e749f4b41244a89bb852afdbabd